src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'redux/books/ADD_BOOK';
+const REMOVE_BOOK = 'redux/books/REMOVE_BOOK';
+
+const reducer = (initialState = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...initialState, action.payload];
+    case REMOVE_BOOK:
+      return [...initialState.filter((book) => (book.id !== action.id))];
+    default:
+      return initialState;
+  }
+};
+
+const createActionForBookToAdd = (title, author) => ({
+  type: ADD_BOOK,
+  payload: {
+    title,
+    author,
+  },
+});
+
+const createActionForBookToRemove = (id) => ({
+  type: REMOVE_BOOK,
+  payload: id,
+});
+
+export default reducer;
+export { createActionForBookToAdd, createActionForBookToRemove };
```

Comment:
- [ ] Here in this file, you need to import createSlice from redux toolkit as it is a requirement and also a better and newer way to create reducers and action creators. kindly take guidance on how to do that from this [website](https://redux-toolkit.js.org/api/createSlice). it will change the way you export reducers from this file and then you are going to import those reducers in configureStore.js file and use them there with configureStore from the redux toolkit.
- [ ] When you use createSlice, you will be taking initialState, addBook and removeBook reducer, all of that inside bookSlice, fulfilling these requirements:
   - Define a slice of state for books that:
      - Includes an array of books (initial state: empty array)
      - Includes a reducer that adds a book
      - Includes a reducer that removes a book